eng/ApiListing.targets

```diff
@@ -20,7 +20,13 @@
       <GenAPIAdditionalParameters>--exclude-attributes-list "$(MSBuildThisFileDirectory)ApiListing.exclude-attributes.txt" $(GenAPIAdditionalParameters)</GenAPIAdditionalParameters>
       <GenAPITargetPath>$(_RefSourceOutputPath)$(AssemblyName).$(TargetFramework).cs</GenAPITargetPath>
     </PropertyGroup>
-    <Delete Files="$(GenAPITargetPath)" />
+    <ItemGroup>
+      <!-- Remove all files that don't start with $(AssemblyName) and the current target -->
```

Comment:
Fixes an issue where copying Azure.Template and re-running Export-API leaves the original API listing around and confuses people.